process/election-schedule.md

```diff
@@ -0,0 +1,36 @@
+**TAC Election Schedule v1.2**
```

Comment:
This probably shouldn't be here, it's covered in the OpenSSF charter

Reply:
I think this should be updated to reflect the current status of the OpenSSF's TAC - a lot of this is covered in the current charter, yes, but there are also details, like election results, which are better captured (and then shared to the community) here.